python-client/giskard/ml_worker/testing/tests/metamorphic.py

```diff
@@ -57,12 +57,16 @@ def _compare_prediction(results_df, prediction_task, direction, output_sensitivi
                 axis=1,
             )
             passed_idx = results_df.loc[results_df["predict_difference_ratio"] < output_sensitivity].index.values
+        else:
+            raise ValueError(f"Invalid prediction task: {prediction_task}")
```

Comment:
any reason to prefer manual check over defining types and adding `@configured_validate_arguments`